lib/array_equals.rb

```diff
@@ -1,5 +1,18 @@
-# Determines if the two input arrays have the same count of elements
+# Determines if the two input arrays have the same count of elecdments
 # and the same integer values in the same exact order
 def array_equals(array1, array2)
-  raise NotImplementedError
+
```

Comment:
You also need to check if array1 and array2 are nil.

```suggestion
return true if array1.nil?  && array2.nil?
return false if array1.nil?  || array2.nil?
```